tuner/tuner/candidate_gen_test.py

```diff
@@ -48,13 +48,25 @@ def test_apply_params_mmt(tuner_ctx: common.TunerContext) -> None:
 
     mma_intrinsic = iree_gpu.MMAIntrinsic.MFMA_F32_16x16x16_F16
     mma_attr = iree_gpu.MMAAttr.get(mma_intrinsic)
+    lowering_config_dict = {
+        "mma_kind": mma_attr,
+        "workgroup": ir.ArrayAttr.get(
+            [
+                ir.IntegerAttr.get(tuner_ctx.type.i32, 8),
+                ir.IntegerAttr.get(tuner_ctx.type.i32, 8),
+                ir.IntegerAttr.get(tuner_ctx.type.i32, 8),
+            ]
+        ),
+        "reduction": ir.ArrayAttr.get([]),
+        "subgroup_m_count": ir.IntegerAttr.get(tuner_ctx.type.i32, 16),
+        "subgroup_n_count": ir.IntegerAttr.get(tuner_ctx.type.i32, 16),
+    }
+
+    lowering_config_attrs = ir.DictAttr.get(lowering_config_dict)
```

Comment:
We can create a helper function that accepts python types and returns a lowering config attribute to avoid having to create these dictionaries by hand.

For example, this could be something like: `common.getLoweringConfig(mma_kind=mma_attr, workgroup=[8,8,0], reduction=[0, 0, 8], subgroup_m_count=16, subgroup_n_count=16)`.

Reply:
This could also be new method defined in python bindings, but I think it will be easier to define this in python as a new helper.